src/test/java/com/glencoesoftware/omero/ms/image/region/ImageRegionCtxTest.java

```diff
@@ -562,13 +581,13 @@ public void testMapsLogarithmic() {
         ctx.setMapProperties(renderer, families, 0);
         ChannelBinding cb = renderer.getChannelBindings()[0];
         Assert.assertEquals(cb.getFamily().getValue(), "logarithmic");
-        Assert.assertEquals(cb.getCoefficient(), 1.8);
+        Assert.assertEquals(cb.getCoefficient(), 1.0);
         Assert.assertEquals(cb.getNoiseReduction(), new Boolean(false));
     }
 
-    @Test(expectedExceptions = IllegalArgumentException.class)
-    public void testMapsLogarithmicNegativeCoefficient() {
-        String maps = "[{\"reverse\": {\"enabled\": false}, \"quantization\" :{\"family\":\"logarithmic\",\"coefficient\":-1.8}}]";
+    @Test
```

Comment:
Do you really want to loose your out of range coefficient test?

Reply:
It's not so much about losing it, but if the coefficient never made sense for logarithmic stuff in the first place, then such a test at least shouldn't be here. I wasn't sure if there are illegal coefficients for other quantization families

Reply:
There are. < 0 for example, which as you were using -1.8 previously and were specifically checking negative coefficients was why I was raising it.